### DISCUSSION TITLE
disallow blank lines in if statement

### DISCUSSION THREAD

**Discussion by LangLangBart** [eslint/eslint#16774] [Q&A] [answered]
Hello, is there any rule that can be used to get rid of an empty line as seen in the example below?

### given example
```js
if (a)

	b();
```

### intended result
```js
if (a)
	b();
```

### experimented with these `eslint` rules rules, no success
- [padding-line-between-statements](https://eslint.org/docs/latest/rules/padding-line-between-statements)
- [padded-blocks](https://eslint.org/docs/latest/rules/padded-blocks)

---
Technically, this would work, but it should not be applied to the whole document, just within the if statement.

```json
      "no-multiple-empty-lines": [
        "error",
        {
          "max": 0
        }
      ],
```

**Comment by mdjermanovic** [comment, marked as answer]
No, there is no core rule that would disallow empty lines after `if (a)`.

`padding-line-between-statements` requires/disallows empty lines between statements in statement lists, but not between parent-child statements. `padded-blocks` requires/disallows empty lines after `{` and before `}`.
